modules/web-demo/webpack/base.config.js

```diff
@@ -88,6 +88,10 @@ const rules = [
       emitFile: false,
     },
   },
+  {
+    test: /miniscript.*\.wasm$/,
+    type: 'webassembly/sync',
```

Comment:
For some reason `webassembly/async` does not work here. The import stays in some sort of deferred state.